### DISCUSSION TITLE
Renaming Folders is no longer available

### DISCUSSION THREAD

**Discussion by mkornegay2** [iterate-ch/cyberduck#16702] [Q&A] [answered]
I just updated Cyberduck and now I cannot rename my folders. I'm locked out of my website due to a plugin and renaming the folder to deactivate it was how I got in. Now I cannot do this.

**Comment by dkocher** [comment, marked as answer]
We are aware of the regression [^1] and are working on a fix.

[^1]: #16695

**Reply by mkornegay2** [reply]
Uhhh... I really need to get access to my website. And Wordfence has locked me out.

**Reply by TomHsiung** [reply]
Same issue. Rename option is in gray.

**Reply by Zyron** [reply]
Not just folder, also files.

**Reply by amochkin** [reply]
Is there a way to download and install previous version?

**Reply by dkocher** [reply]
Update to the current [snapshot build](https://docs.cyberduck.io/cyberduck/preferences/#snapshot-builds) which includes the fix.
